src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -272,12 +247,26 @@ suspend fun CoroutineScope.launchListeners(
             blockingApplicationLogic(applicationState, kafkaconsumerRecievedSykmelding, kafkaproducerCreateTask,
                     kafkaproducervalidationResult, infotrygdOppdateringProducer, infotrygdSporringProducer,
                     session, personV3, arbeidsfordelingV1, env.sm2013BehandlingsUtfallToipic, norskHelsenettClient,
-                    smIkkeOkQueue, norg2Client, jedis)
+                    smIkkeOkQueue, norg2Client, jedis, kafkaproducerreceivedSykmelding, env.sm2013infotrygdRetry)
+        }
+    }.toList()
+
+    val recievedSykmeldingretryListeners = 0.until(env.applicationThreads).map {
```

Comment:
Ønsker vi egentlig å kjøre flere lyttere pr pod? Synes vi bør forenkle dette til en lytter pr topic pr pod, dvs en lytter for retry og en for den vanlige topicen pr pod :)

Reply:
ja kanskje. Veit ikkje heilt hvordan eg skal løse det med tanke på lytter pr topic pr pod.

Reply:
Du kan bare fjerne den funksjonaliteten her som lager flere..? Samme som jeg gjorde i papirmottaket :)